### PR TITLE
Add sendgrid handler

### LINKED ISSUE #4: Sendgrid Handler
We can use an HTTP client to send to sendgrid logs instead of mailing them

### PR DESCRIPTION
**Pull request by gmponos**
closes #4

### CODE REVIEW

File: src/SendGridHandler.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        $body = $request->getBody();
+        $body->write(\http_build_query($message));
+        $body->rewind();
+
+        return $request;
+    }
+
+    private function isHtmlBody(string $body): bool
+    {
+        return \substr($body, 0, 1) === '<';
```
Comment (savvasal) [inline src/SendGridHandler.php]: Let's use
`$body !== strip-tags($body)`

File: tests/SendGridHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                ['to@domain.com'],
+                'There was an error'
+            )
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function createRequest(): void
```
Comment (savvasal) [inline tests/SendGridHandlerTest.php:49]:
```suggestion
    public function testCreateRequest(): void
```

**Review by savvasal — APPROVED** [review]

File: tests/SendGridHandlerTest.php (hunk 2 above)
Comment (gmponos) [reply tests/SendGridHandlerTest.php:49]: We are using a convention from php-cs-fixer

File: src/SendGridHandler.php (hunk 1 above)
Comment (gmponos) [reply src/SendGridHandler.php]: nice.

**Pull request merged**